### PR TITLE
Change mapping and aliases for NIH and CB

### LINKED ISSUE #71: Change mapping and aliases for NIH and CB
Some fields would be better off as `text` to be able to search in them:
- places in general
- websites (excluding social media as these usually have IDs or nicknames in URL, TBD)
- categories: a company might be categorised as "medical devices" and ES (or Clio?) might rank it higher when we are searching for "medicine".

We need a `state_id` for NIH too.

`countries_codes` should be `countries_ids` to be consistent with a single `countries_id` (and it's shorter).

### PR DESCRIPTION
**Pull request by mindrones**
- [x] Rename 'countries_codes', add 'state_id'
- [x] change some fields type from keyworkd -> text
- [x] sort orms configs alphabetically helps reading
- [x] check the [array fields](https://www.elastic.co/guide/en/elasticsearch/reference/6.6/array.html)
- [ ] squash

Closes #71

### CODE REVIEW

**Comment by jaklinger** [comment]
sitting on this review until after the new data processing has been checked

**Comment by mindrones** [comment]
See https://github.com/nestauk/nesta/issues/81#issuecomment-498218507

File: nesta/production/orms/nih_es_config.json [hunk 1]
```diff
@@ -86,31 +58,80 @@
...
                             "type": "keyword"
                         }
-                    },
-                    "analyzer": "mesh_terms_analyzer"
+                    }
                 },
-                "booleanFlag_duplicate_abstract": {
-                    "type": "boolean",
-                    "null_value": false
+                "placeName_continent_organisation": {
```
Comment (jaklinger) [inline nesta/production/orms/nih_es_config.json]: continents are single words, so I guess this is redundant

Comment (mindrones) [reply nesta/production/orms/nih_es_config.json]: The reason I added `text`here is that if I got it right with `keyword` looking for "asia" won't find ["Asia"](https://nesta-open-data.s3.eu-west-2.amazonaws.com/rwjf-viz/iso_a2_to_name_by_type.json), correct?

Comment (jaklinger) [reply nesta/production/orms/nih_es_config.json]: good point!

Comment (jaklinger) [reply nesta/production/orms/nih_es_config.json]: oh wait, no your logic is the wrong way around: `keyword` is case insensitive so "Asia" and "asia" will both return "Asia"

Comment (mindrones) [reply nesta/production/orms/nih_es_config.json]: Ah I see, good to know! Then I'd say `text` can go

**Comment by jaklinger** [comment]
ok to close this?

**Comment by mindrones** [comment]
yeps 👍

**Pull request closed without merging**